### PR TITLE
Clean some variables/code

### PR DESCRIPTION
**Pull request by aleeks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Clean some variables/code
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | looks the code :p

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

### CODE REVIEW

**Review by LittleBigDev — APPROVED** [review]

**Review by eternoendless — APPROVED** [review]
Code looks good!

**Comment by eternoendless** [comment]
Thanks @aleeks

File: controllers/admin/AdminAddressesController.php [hunk 1]
```diff
@@ -444,12 +444,6 @@ protected function processAddressFormat()
 
         $selected_country = ($tmp_addr && $tmp_addr->id_country) ? $tmp_addr->id_country : (int)Configuration::get('PS_COUNTRY_DEFAULT');
 
-        $inv_adr_fields = AddressFormat::getOrderedAddressFields($selected_country, false, true);
-        $dlv_adr_fields = AddressFormat::getOrderedAddressFields($selected_country, false, true);
-
-        $inv_all_fields = array();
-        $dlv_all_fields = array();
```
Comment (Quetzacoalt91) [inline controllers/admin/AdminAddressesController.php:451]: NOPE ! You removed variables later used in this function !!!!

**Review by Quetzacoalt91 — COMMENTED** [review]
PR needs to be partially reverted

File: controllers/admin/AdminAddressesController.php (hunk 1 above)
Comment (LittleBigDev) [reply controllers/admin/AdminAddressesController.php:451]: IMHO this method is unclear. Maybe efficient and concise but really difficult to read and understand. This led @aleeks to remove actually used vars.
Maybe this is a good time to refacto this method.

Comment (eternoendless) [reply controllers/admin/AdminAddressesController.php:451]: Dammit, you're right! My bad. This is why dynamic variable names should be avoided.

**Pull request merged**